tools/c7n_gcp/c7n_gcp/resources/spanner.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from c7n_gcp.provider import resources
+from c7n_gcp.query import QueryResourceManager, TypeInfo
+
+
+@resources.register('spanner-instance')
+class SpannerInstance(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = 'spanner'
+        version = 'v1'
+        component = 'projects.instances'
+        enum_spec = ('list', 'instances[]', None)
+        scope_key = 'parent'
+        scope_template = 'projects/{}'
+        id = 'name'
+
+        @staticmethod
+        def get(client, resource_info):
+            return client.execute_command(
+                'get', {'name': resource_info['name']}
+            )
+
+
+@resources.register('spanner-instance-config')
+class SpannerInstanceConfig(QueryResourceManager):
```

Comment:
why is this is a resource, it seems like its data center listing, not configurable or useful.

Reply:
Rechecked the resource. It's not useful itself nor there are spanner resources with a location dependency (argument). Removed. Thanks.